swiggy/src/main/kotlin/lld/swiggy/models/OrderModels.kt

```diff
@@ -4,8 +4,9 @@ data class Order(
     val orderId: EntityPublicId,
     val userId: EntityPublicId,
     val restaurantId: EntityPublicId,
```

Comment:
why is everything of type `EntityPublicId`

Reply:
just defining a general public id for every entity. Otherwise, it's just `String`

Reply:
makes sense actually, I will create this as an interface that would be implemented by individual classes like UserId, and RestaurantId. in the next PR